sgrpg.js

```diff
@@ -59,11 +60,49 @@ Hooks.once("init", function(){
         config: true
     });
 
+    game.settings.register("sgrpg", "showDescriptionDefault", {
+        name: "Show Item Description",
+        hint: "Whether to show the item description by default.",
+        scope: "world",
+        type: Boolean,
+        default: false,
+        config: true
+    });
+
+    game.settings.register("sgrpg", "useMoxieCombat", {
+        name: "Use Moxie in Combat",
+        hint: "Whether to use Moxie instead of Initiative in combat.",
+        scope: "world",
+        type: Boolean,
+        default: false,
+        config: true
+    });
+
+    game.settings.register("sgrpg", "autoLevelSystem", {
+        name: "Calculate level from MP",
+        hint: "Whether the system will automatically calculate a character's level from their spent MP, given in the 'Spent MP' field.",
+        scope: "world",
+        type: Boolean,
+        default: false,
+        config: true
+    });
 
-    preloadHandlebarsTemplates()
+    // Pure homebrew
+    game.settings.register("sgrpg", "allowWeaponTensionOnAttack", {
+        name: "Homebrew: Weapon Tension Die can be set for attack rolls",
+        hint: "Homebrew setting: Whether the system allows the burst and automatic fire tension dice to be rolled for the attack roll instead of the damage roll.",
+        scope: "world",
+        type: Boolean,
+        default: false,
+        config: true
+    });
```

Comment:
I would prefer this to be some external plugin ideally. Not sure how much work that would be though. Having it in default system though could lead to not so nice precedence for more homebrew rules that i feel we should not introduce